reference/manual-upgrade.md

```diff
@@ -0,0 +1,391 @@
+
+**RPC:**
+  - Control Plane: The RCP for the control plane has been changed from NATS to gRPC.
+  - Data Plane: The registration heartbeat has been changed from NATS to gRPC. 
+
+**Pool CRDs:**
+  - The pool CRDs have been renamed `DiskPools` (previously, MayastorPools).
+  {% endhint %}
+
+
+
+1. In order to start the upgrade process, the following previously deployed components have to be deleted. 
+
+ - To delete the control-plane components, execute: 
+    {% tab title="Commands" %}
+    ```text
+    kubectl delete deploy core-agents -n mayastor
+    kubectl delete deploy csi-controller -n mayastor
+    kubectl delete deploy msp-operator -n mayastor
+    kubectl delete deploy rest -n mayastor
+    kubectl delete ds mayastor-csi -n mayastor
+    ```
+    {% endtab %}
+ 
+- Next, delete the associated RBAC operator. To do so, execute:
+    {% tab title="Commands" %}
+    ```text
+    kubectl delete -f https://raw.githubusercontent.com/openebs/mayastor-control-plane/<version>/deploy/operator-rbac.yaml
+    ```
+    {% endtab %}
+
+{% hint style="info" %}
+In the above command, add the previously installed Mayastor version in the format v1.x.x
+{% endhint %}
+
+2. Once all the above components have been successfully removed, pull the latest helm chart from [Mayastor-extension repo](https://github.com/openebs/mayastor-extensions) and save it to a file, say `helm_templates.yaml`. To do so, execute:
+
+   {% tab title="Command" %}
+    ```text
+    helm template mayastor . -n mayastor --set etcd.persistence.storageClass="manual" --set loki-stack.loki.persistence.storageClassName="manual" --set etcd.initialClusterState=existing > helm_templates.yaml
+    ```
+    {% endtab %}
+
+- Next, update the `helm_template.yaml` file, add the following helm label to all the resources that are being created.
+
+    {% tab title="Helm label" %}
+    ```text
+    metadata: 
+      annotations: 
+        meta.helm.sh/release-name: $RELEASE_NAME 
+        meta.helm.sh/release-namespace: $RELEASE_NAMESPACE 
+      labels: 
+        app.kubernetes.io/managed-by: Helm
+    ```
+    {% endtab %}
+
+- Copy the `etcd` and `io-engine` spec from the `helm_templates.yaml` and save it in two different  files say, mayastor_2.0_etcd.yaml and mayastor_io_v2.0.yaml. Once done, remove the `etcd` and `io-engine` specs from `helm_templates.yaml`. These components need to be upgraded separately.  
+
+3.  Install the new control-plane components using the `helm-templates.yaml` file. 
+
+    {% tab title="Command" %}
+    ```text
+    kubectl apply -f helm_templates.yaml -n mayastor
+    ```
+    {% endtab %}
+
+    {% hint style="info" %}
+    In the above method of installation, HA is disabled by default. The steps to enable HA are described later in this document.
```

Comment:
the nexus (target) High Availability (HA) is disabled by default.

Reply:
Updated the text.